test/contracts-network/meta-colony.js

```diff
@@ -565,48 +564,6 @@ contract("Meta Colony", (accounts) => {
     });
   });
 
-  describe("when setting the metaColony stipend", () => {
-    it("should allow the metacolony stipend to be set", async () => {
-      const stipendBefore = await colonyNetwork.getAnnualMetaColonyStipend();
-      expect(stipendBefore).to.be.zero;
-      await metaColony.setAnnualMetaColonyStipend(WAD);
-      const stipendAfter = await colonyNetwork.getAnnualMetaColonyStipend();
-      expect(stipendAfter).to.eq.BN(WAD);
-    });
-
-    it("setting the metacolony stipend should be a permissioned function", async () => {
-      await checkErrorRevert(metaColony.setAnnualMetaColonyStipend(0, { from: OTHER_ACCOUNT }), "ds-auth-unauthorized");
-    });
-
-    it("a non-meta colony should not be able to set the stipend", async () => {
-      ({ colony } = await setupRandomColony(colonyNetwork));
-      const colonyAsMetaColony = await IMetaColony.at(colony.address);
-      await checkErrorRevert(colonyAsMetaColony.setAnnualMetaColonyStipend(0), "colony-caller-must-be-meta-colony");
-    });
-
-    it("should allow the metacolony stipend to be claimed, and the amount minted changes based on when it is last claimed", async () => {
-      await metaColony.setAnnualMetaColonyStipend(WAD);
-      // In a year, we should get the full reward
-      let time = await currentBlockTime();
-      time = new BN(time).addn(SECONDS_PER_DAY * 365);
-      await makeTxAtTimestamp(colonyNetwork.issueMetaColonyStipend, [], time.toNumber(), this);
-      let potBefore = await metaColony.getFundingPotBalance(1, clnyToken.address);
-      await metaColony.claimColonyFunds(clnyToken.address);
-      let potAfter = await metaColony.getFundingPotBalance(1, clnyToken.address);
-      expect(potAfter.sub(potBefore)).to.eq.BN(WAD);
-      // If we claim again a minute later...
-      potBefore = potAfter;
-      await makeTxAtTimestamp(colonyNetwork.issueMetaColonyStipend, [], time.addn(60).toNumber(), this);
-      await metaColony.claimColonyFunds(clnyToken.address);
-      potAfter = await metaColony.getFundingPotBalance(1, clnyToken.address);
-      expect(potAfter.sub(potBefore)).to.eq.BN(WAD.divn(365 * 24 * 60));
-    });
-
-    it("the metacolony stipend cannot be claimed before it is set", async () => {
-      await checkErrorRevert(colonyNetwork.issueMetaColonyStipend(), "colony-network-metacolony-stipend-not-set");
-    });
-  });
-
```

Comment:
I was expecting to see these tests rewritten so that they used the extension, rather than just being deleted outright :cry:

Reply:
I don't think thats necessary -- the functionality is tested in the token-supplier tests, and there's nothing fundamentally unique about using the extension in the metaColony. The tests would be logically redundant.

Reply:
My worry here was around subtle differences in functionality or permissions that we would want to be double checking that we were happy with. After looking at this for a while, the only thing I can see is that we have to declare an upper limit for the supply, which I guess is fine.